### DISCUSSION TITLE
PyVista

### DISCUSSION THREAD

**Discussion by labriki** [gempy-project/gempy#858] [Q&A] [answered]
In the tutorials I didn't see:
1 - How to print the model displayed in pyvista;
2 - How to export it to view it in other applications;
3 - How to change the size of the displayed fonts.

**Comment by AlexanderJuestel** [comment]
@labriki,

Please have a look at our GemGIS package and in particular the following tutorial that allows extracting the surfaces as pyvista mesh. You can use pyvista then to plot these meshes (including changing the font sizes) and saving them as vtk files for other applications.

Let us know if that helped you out :)

https://gemgis.readthedocs.io/en/latest/getting_started/tutorial/18_creating_depth_maps_from_gempy_models.html

**Reply by labriki** [reply]
Thinks, 
Can i install GamEs in the same environment as Gempy?

**Reply by AlexanderJuestel** [reply]
Yes, that should cause no issues

**Reply by labriki** [reply, marked as answer]
Thank you very much, I managed to get the tutorial to work and noticed that it loads the data for me. This is excellent, it will allow me to see the raw data. well done, it’s impressive.
I have the vtk files how to view them better apart from the image in the notebook?
